lib/spack/spack/directory_layout.py

```diff
@@ -348,6 +350,13 @@ def remove_install_directory(self, spec, deprecated=False):
         """
         path = self.path_for_spec(spec)
         assert(path.startswith(self.root))
+        kwargs = {'ignore_errors': True}
```

Comment:
Apologies I just realized: unlike `remove_linked_tree`, the invocation of `shutil.rmtree` did not ignore errors originally, so this should probably be more like:

```
if is_windows:
  kwargs = {ignore_errors: false, onerror: fs.error_remove_readonly}
else:
  kwargs= {} # the default value for ignore_errors is false
```

does that seem reasonable?

Reply:
> Apologies I just realized: unlike remove_linked_tree, the invocation of shutil.rmtree

I meant: "__this__ invocation of shutil.rmtree" i.e. as it's being called in this function